implementation/src/main/java/io/smallrye/mutiny/Multi.java

```diff
@@ -471,6 +471,33 @@ default <O> Multi<O> concatMap(Function<? super T, ? extends Publisher<? extends
         return onItem().transformToMultiAndConcatenate(mapper);
     }
 
+    /**
+     * Produces a {@link Multi} containing the items from {@link Publisher} produced by the {@code mapper} for each
+     * item emitted by this {@link Multi}.
+     * <p>
+     * The operation behaves as follows:
+     * <ul>
+     * <li>for each item emitted by this {@link Multi}, the mapper is called and produces a {@link Publisher}
+     * (potentially a {@code Multi}). The mapper must not return {@code null}</li>
+     * <li>The items emitted by each of the produced {@link Publisher} are then <strong>concatenated</strong> in the
+     * produced {@link Multi}. The flatten process makes sure that the items are not interleaved.
+     * If {@code prefetch} is set to {@code true}, additional item is prefetched from the upstream, otherwise items
+     * are requested lazily from the upstream, one at a time.
+     * </ul>
+     * <p>
+     * This method is equivalent to {@code multi.onItem().transformToMulti(mapper).concatenate(prefetch)}.
+     *
+     * @param mapper the {@link Function} producing {@link Publisher} / {@link Multi} for each items emitted by the
+     *        upstream {@link Multi}
+     * @param prefetch whether the operator prefetches items from upstream, or not.
+     * @param <O> the type of item emitted by the {@link Publisher} produced by the {@code mapper}
+     * @return the produced {@link Multi}
+     */
+    @CheckReturnValue
+    default <O> Multi<O> concatMap(Function<? super T, ? extends Publisher<? extends O>> mapper, boolean prefetch) {
```

Comment:
I'm wondering if this shortcut is useful. Just having the possibility to configure the prefetch in `onItem().transformToMultiAndConcatenate()` may be enough.